osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -473,5 +506,43 @@ private void load()
                 Loading = true;
             }
         }
+
+        private class MutedNotification : SimpleNotification
+        {
+            public MutedNotification()
+            {
+                Text = "Your music volume is set to 0%! Click here to restore it.";
+            }
+
+            public override bool IsImportant => true;
+
+            public override bool RequestsFocus => true;
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours, AudioManager audioManager, NotificationOverlay notificationOverlay, VolumeOverlay volumeOverlay)
+            {
+                Icon = FontAwesome.Solid.VolumeMute;
+                IconBackgound.Colour = colours.RedDark;
+
+                Activated = delegate
+                {
+                    notificationOverlay.Hide();
+
+                    volumeOverlay.IsMuted.Value = false;
+                    audioManager.Volume.SetDefault();
+                    audioManager.VolumeTrack.SetDefault();
+
+                    return true;
+                };
+            }
+        }
+
+        /// <summary>
+        /// Sets <see cref="muteWarningShownOnce"/> to false, reserved for testing.
+        /// </summary>
+        public static void ResetNotificationLock()
```

Comment:
This function should go away. instead, you should resolve the `SessionStatics` in the test and reset it.